test/PaginationSpec.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ReactTestUtils from 'react/lib/ReactTestUtils';
+import Pagination from '../src/Pagination';
+
+describe('Pagination', function () {
+  it('Should have class', function () {
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Pagination>Item content</Pagination>
+    );
+    assert.ok(ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'pagination'));
+  });
+
+  it('Should show the correct active button', function () {
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Pagination
+        items={5}
+        activePage={3} />
+    );
+    let pageButtons = ReactTestUtils.scryRenderedDOMComponentsWithTag(instance, 'li');
+    assert.equal(pageButtons.length, 5);
+    assert.equal(pageButtons[2].getDOMNode().getAttribute('class').match(/active/)[0], 'active');
```

Comment:
`React.findDOMNode` is more future proof than `getDOMNode()` We still have a number of tests using it but if we can use the newer function that would help us out.